src/main/java/com/jfrog/ide/idea/ui/configuration/JFrogGlobalConfiguration.java

```diff
@@ -274,6 +284,12 @@ private void loadConfig() {
             watches.setText(serverConfig.getWatches());
             connectionRetries.setValue(serverConfig.getConnectionRetries());
             connectionTimeout.setValue(serverConfig.getConnectionTimeout());
+            if (!StringUtils.isEmpty(serverConfig.getExternalResourcesRepo())) {
+                useExternalRepositoryCheckBox.setSelected(true);
+                repositoryNameJBTextField.setText(serverConfig.getExternalResourcesRepo());
+            } else {
+                useExternalRepositoryCheckBox.setSelected(false);
+            }
```

Comment:
We can remove the select box and have only the text box. It complicates the code. Here, we need to check if there is a text and then set the checkbox accordingly.

Reply:
It was changed to radio buttons. For a better UX, we decided to add these radio buttons in addition to the textbox, to make it clearer to the user what happens if the textbox remains empty.